static_frame/test/unit/test_frame.py

```diff
@@ -6160,9 +6160,32 @@ def test_frame_to_latex_a(self) -> None:
                 \end{table}
                 '''
         self.assertEqualLines(post, msg)
-
-
-
+    
+    def test_frame_to_msgpack_a(self) -> None:
+        records = (
+                (2, 'a', False),
+                (3, 'b', False),
+                )
+        f1 = Frame.from_records(records,
+                columns=('r', 's', 't'),
+                index=('w', 'x'))
```

Comment:
If we have integer labels here will they come back as integers or strings?

Reply:
I'll add more test coverage. Also realized last night I forgot to add tests for things like np.float_64 and np.int64, which was the whole reason for involving msgpack_numpy.

Reply:
Better tests added covering:
int
np.float64
np.short
np.int64
np.datetime64

see :
test_frame_from_msgpack_b
test_frame_from_msgpack_c

Reply:
Very nice expansion of the test coverage! When you want to go even further, you can try implementing a property test with Hypothesis (https://hypothesis.readthedocs.io). SF has a library of Hypothesis strategies that "randomly" generate incredibly diverse ``Frame`` and ``Series`` for testing. Take a look at `test.property.test_frame.test_frame_to_xlsx` to see an example. This will inevitable shake out edge cases and types you were not expecting.

Reply:
I definitely want to read up on that. This is the first time I've ever encountered that library. So many fun new concepts ^-^